types/SupabasePost.ts

```diff
@@ -11,4 +11,5 @@ export type SupabasePost = {
   degen?: number;
   parent_url?: string;
   channelId?: string;
+  id:number
```

Comment:
is this ID coming from supabase?

Reply:
if yes, which file should I reference to verify where `id` is being pulled from supabase?